packages/flutter_tools/test/commands.shard/hermetic/daemon_test.dart

```diff
@@ -300,16 +307,185 @@ void main() {
       FuchsiaWorkflow: () => FakeFuchsiaWorkflow(),
     });
 
+    testUsingContext('device.discoverDevices should respond with list', () async {
```

Comment:
Not a blocker, but I'm curious to know whether these tests touch the global context. Could they be `testWithoutContext`?

Reply:
The `Daemon` class in `commands/daemon.dart` is using the global context in several places, it errors out when I tried to use `testWithoutContext`.